src/main/java/Suv.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Scanner;
+
+
+public class Suv {
+    final static int DEADLINE_KEYWORD_END_INDEX = 8;
+    final static int TODO_KEYWORD_END_INDEX = 4;
+    final static int EVENT_KEYWORD_END_INDEX = 5;
+    final static int TO_KEYWORD_END_INDEX = 2;
+    final static int FROM_KEYWORD_END_INDEX = 4;
+
+    public static void main(String[] args) {
+        String name = "Suv";
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskIndex = 0;
+
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm " + name + "\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n"
+        );
+
+        while(true) {
+            String input = in.nextLine();
+            if(input.equals("bye")){
+                System.out.println( "____________________________________________________________\n" +
+                        " Bye. Hope to see you again soon!\n" +
+                        "____________________________________________________________\n"
+                );
+                break;
+            } else if (input.equals("list")){
+                System.out.println("____________________________________________________________\n" + " Here are the tasks in your list:");
```

Comment:
maybe you could abstract the line printing portion to a short method to shorten the lines for better readability